docs/source/releases/1.17.2/1.17.2-bug-fixes.yaml

```diff
@@ -0,0 +1,29 @@
+bug fix:
+- title: 'Assorted minor bug fixes.'
+  description: |
+    * 1.16.0 release note had start after finish date, broke build_docs.sh.
+    * sector resolution should be number not integer in jsonschema
+    * Include 'not disabled' by default in all interactive_pytest.sh commands
+
+      * Explicitly request 'disabled' tests.
+    * Minor change to the 3 geotiff outputs (not visible to the naked eye, amsr2, ami,
```

Comment:
I'm pretty sure I update the same files multiple times,  I expect we'll need to update the geotiff output checker to make sure it is not comparing unimportant header information with the comparisons